libraries/tools/kotlin-stdlib-gen/src/templates/Aggregates.kt

```diff
@@ -846,7 +860,12 @@ object Aggregates : TemplateGroupBase() {
         inline()
         specialFor(ArraysOfUnsigned) { inlineOnly() }
 
-        doc { "Accumulates value starting with last ${f.element} and applying [operation] from right to left to each ${f.element} and current accumulator value." }
+        doc {
+            """
+            Accumulates value starting with last ${f.element} and applying [operation] from right to left to each ${f.element} and current accumulator value.
+            @throws UnsupportedOperationException if the ${f.collection} is empty.
```

Comment:
As per the migration plan of https://youtrack.jetbrains.com/issue/KT-33874 we're going to document in 1.4 only that an exception is thrown in that case, without specifying its concrete class.

We can also advise not to catch it, but use `reduceOrNull` alternative instead which is going to be introduced soon (https://youtrack.jetbrains.com/issue/KT-33761).